junebug/tests/test_command_line.py

```diff
@@ -509,7 +504,7 @@ def test_logging_setup(self):
         is not None, both the stdout logger and a file logger is created'''
         logging_setup(None, None)
         [handler] = logging.getLogger().handlers
-        self.assertEqual(handler.stream.name, '<stdout>')
+        self.assertTrue(handler.stream.name in ['<stdout>', '<fdopen>'])
```

Comment:
This change allows us to use `py.test -x -f` during development. Py tests replaces <stdout> with a fd for capturing test log output